iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -128,9 +128,10 @@ public enum ErrorCode
         JobQuotaExceeded = 403003,
 
         /// <summary>
-        /// The underlying cause is that the number of messages enqueued for the device exceeds the queue limit (50).
-        /// The most likely reason that you're running into this limit is because you're using HTTPS to receive the message,
-        /// which leads to continuous polling using ReceiveAsync, resulting in IoT hub throttling the request.
+        /// The underlying cause is that the number of cloud-to-device messages enqueued for the device exceeds the queue limit (50).
```

Comment:
Do we want to include a real number here? What if this changes?

In all cases where we have a condition where you are meeting or exceeding some hard limit defined by the service we should refer to the documentation for this number.

The only exception would be when we add a limitation in code and then we should link to the property/setting that controls it.

Reply:
Best refer to service documentation